### ISSUE TITLE
Display Duration in the Most Watched Component Promo

### ISSUE THREAD

**Issue by jroebu14** [bbc/simorgh#7630]
**Is your feature request related to a problem? Please describe.**

__As a ws user, Maria,__
I want to know how long a piece of audio or video is 
so that I can decide whether I have the time to watch or listen to it 

__As a ws screenreader user, Carlos,__
I want to know whether a piece of content is a video or a piece of audio
so that I can make an informed decision as to whether I consume it 

---

**Describe the solution you'd like**

__Acceptance criteria:__
The most watched promo will display the clip duration where duration is present  
The most watched h3 label announces duration to screenreaders 
The most watched h3 label announces "video" where the clip is a video to screenreaders 
The most watched h3 label announces "listen" where the clip is audio to screenreaders

__Video Promo__

- Given I am using voiceover on IOS 
- when I navigate to the h3 link within the most watched promo list
- and the link is to a MAP with a video
- then the screenreader announces “video, 'headline', duration 'duration time', heading level 3, link” (e.g. “video, “News of the Day”, duration 05,00, heading level 3, link”) (semantic role order depends on the assistive tech)
- and "video", "duration" are announced in the language of the service
- and "heading level 3" and "link" are announced in the language recognised by the screenreader

__Audio Promo__

- Given I am using voiceover on IOS 
- when I navigate to the h3 link within the most watched promo list
- and the link is to a MAP with an audio clip 
- then the screenreader announces “Listen, 'headline', duration 'duration time', heading level 3, link” (e.g. “video, “News of the Day”, duration 05,00, heading level 3, link”) (semantic role order depends on the assistive tech)
- and "listen", "duration" are announced in the language of the service
- and "heading level 3" and "link" are announced in the language recognised by the screenreader

---

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

**Comment by LilyL0u** [comment]
@pharingee For audio content, the screenreader reads out 'audio' instead of 'listen'. Is this acceptable according to requirements @aeroplanejane @greenc05 ?

"heading level 3" and "link" are announced in English, not the language of the service.

NVDA
audio 'heading level 3 link, audio, [title], [duration e.g one zero one]'
video 'heading level 3 link, video, [title], [duration e.g one zero one]'

JAWS
audio 'heading level 3 link, audio, [title], [duration]'
video 'heading level 3 link, video, [title], [duration]'

Talkback
audio 'video, [title], [duration], heading 3 link'
video 'video, [title], [duration], heading 3 link'

**Comment by greenc05** [comment]
> "heading level 3" and "link" are announced in English, not the language of the service.

Was the OS in English? If so, that's why.

The screen reader UX should give you the other answers @LilyL0u 

> Talkback
> audio '**video**, [title], [duration], heading 3 link'
> video 'video, [title], [duration], heading 3 link'

Assume you meant audio '**audio**, [title], [duration], heading 3 link'

**Comment by aeroplanejane** [comment]
@greenc05 Shouldn't we say "watch" for video if we're using "listen" for audio?

**Comment by greenc05** [comment]
@aeroplanejane @LilyL0u - Ed specified 'Video' for video in the screen reader UX https://app.zeplin.io/project/5d4d77b1fcbfc79bf47e0294/screen/5f1aebddabaffd369763b5e8

**Comment by aeroplanejane** [comment]
Ok so "listen" for audio and "video" for video - so the acceptance criteria is correct and we need to update "audio" to "listen" @pharingee @LilyL0u Thanks!

**Comment by aeroplanejane** [comment]
Can we ensure that "duration" is announced to screenreaders @pharingee ? Don't think that's currently happening 
 “video, 'headline', duration 'duration time', heading level 3, link”

**Comment by LilyL0u** [comment]
Talkback, JAWS, NVDA - audio is now 'listen'. The word 'duration' in the language is now read out before the numbers of the duration.

Note: on items with no duration, duration is not read out. As we would expect. On items with no alt-text, the image is skipped in the keyboard navigation sequence.
